test/config/validator.js

```diff
@@ -34,6 +34,19 @@ describe('The validator function', () => {
         }).to.throw('No entries found!');
     });
 
+    it('should accept use with copyFiles() only', () => {
+        const config = createConfig();
+        config.setOutputPath('/tmp');
+        config.setPublicPath('/tmp');
+        config.copyFiles({ from: './' });
+
+        !expect(() => {
+            validator(config);
+        }).not.throw();
```

Comment:
I'm not sure that exclamation mark should be there:

```suggestion
        expect(() => { validator(config); }).to.not.throw();
```

Reply:
I don't know why the exclamation mark is there at all :) Thanks for spotting it.